awx/ui_next/src/screens/Template/JobTemplateAdd/JobTemplateAdd.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import { withRouter } from 'react-router-dom';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import {
+  Card,
+  CardBody,
+  CardHeader,
+  PageSection,
+  Tooltip,
+} from '@patternfly/react-core';
+import CardCloseButton from '@components/CardCloseButton';
+import JobTemplateForm from '../shared/JobTemplateForm';
+import { JobTemplatesAPI } from '@api';
+
+function JobTemplateAdd({ history, i18n }) {
+  const [error, setError] = useState('');
+
+  const handleSubmit = async values => {
+    try {
```

Comment:
let's `setError(null)` here (before the try) so the placeholder `error` div goes away on successful resubmits after an error occurs.